foodsaving/users/templates/accountdelete_request-body-text.jinja

```diff
@@ -1,12 +1,13 @@
-{% trans full_name=user.get_full_name(), url=resolve_front_url('delete-account', user.delete_token) %}
+{% load i18n %}
+{% blocktrans with full_name=user.display_name %}
 Hello {{ full_name }},
-we're sad to hear that you want to delete your account on foodsaving.world.
-To finally and irreversibly delete your account click the following link:
+We're sad to hear that you want to delete your account on foodsaving.world.
+To finally and irreversibly delete your account, click the following link:
 
 {{ url }}
```

Comment:
Okay, reverting to Jinja2. Just to be clear, do we have any other uses for templates other than emails? Definitely not to be used for webpages. Pretty limited scope of application in that sense, no?